design/comments.go

```diff
@@ -60,16 +60,10 @@ var createCommentAttributes = a.Type("CreateCommentAttributes", func() {
 })
 
 var commentRelationships = a.Type("CommentRelations", func() {
-	a.Attribute("created-by", commentCreatedBy, "This defines the created by relation")
+	a.Attribute("creator", relationGeneric, "This defines the creator of the comment")
```

Comment:
@kwk issue also talked about changing the name "createdby" to "creator", the db backend change just addresses that.